iptables/iptables.go

```diff
@@ -22,10 +26,33 @@ func AddRule(appPort, metadataAddress, hostInterface, hostIP string) error {
 		return err
 	}
 
-	return ipt.AppendUnique(
-		"nat", "PREROUTING", "-p", "tcp", "-d", metadataAddress, "--dport", "80",
-		"-j", "DNAT", "--to-destination", hostIP+":"+appPort,
-	)
+	proxyRulePresent, proxyRuleLine, err := (*myIPTables)(ipt).detectConcealmentProxyRule(metadataAddress)
+	if err != nil {
+		return err
+	}
+
+	rulespec := []string{"-p", "tcp", "-d", metadataAddress, "--dport", "80",
```

Comment:
While I would like this as well, unfortunately adding the comment would make it incompatible with detecting the current form of the rule and therefore leaving rules behind on update (or additional logic to handle two forms of rules, both for detection and deletion of corresponding one, would have to be introduced). For these resons I would eventually handle this in separate PR. 

(Just to clarify - the rule with comment is reated by GKE, not k8s-gke-service-account-assigner)

Reply:
> (Just to clarify - the rule with comment is created by GKE, not k8s-gke-service-account-assigner)

I know, because I read the comment! :p

Thanks for explaining your reasoning. Your decision makes sense to me.